src/app/lightning/delete-ln-payments.ts

```diff
@@ -43,15 +43,19 @@ const checkAndDeletePaymentForHash = async ({
       const lnPayment = await LnPaymentsRepository().findByPaymentHash(paymentHash)
       if (lnPayment instanceof Error) {
         if (lnPayment instanceof CouldNotFindLnPaymentFromHashError) {
-          const lnd = LndService()
-          if (lnd instanceof Error) return lnd
-          const lnPaymentLookup = await lnd.lookupPayment({ pubkey, paymentHash })
+          // Attempt to get paymentRequest from lnd
+          const lndService = LndService()
```

Comment:
Switched name because we use `lnd` in other parts of the codebase to represent an `AuthenticatedLnd` object